hack/test-go.sh

```diff
@@ -136,7 +136,7 @@ junitFilenamePrefix() {
     return
   fi
   mkdir -p "${KUBE_JUNIT_REPORT_DIR}"
-  local KUBE_TEST_API_NO_SLASH=echo "${KUBE_TEST_API//\//-}"
+  local KUBE_TEST_API_NO_SLASH=$(echo "${KUBE_TEST_API//\//-}")
```

Comment:
do we even need the echo? just `local KUBE_TEST_API_NO_SLASH="${KUBE_TEST_API//\//-}"` might be enough?